### ISSUE TITLE
should test the scheduler class instead of dict

### ISSUE THREAD

**Issue by vincentzlt** [PyTorchLightning/pytorch-lightning#1154]
https://github.com/PyTorchLightning/pytorch-lightning/blob/3ad6169f187ea41aa1534a1d9a3b978d053dca2b/pytorch_lightning/trainer/trainer.py#L711

isinstance(scheduler['scheduler'], optim.lr_scheduler.ReduceLROnPlateau)

**Comment by SkafteNicki** [comment]
PR https://github.com/PyTorchLightning/pytorch-lightning/pull/1126 will fix this

**Comment by SkafteNicki** [comment]
@Borda this can be closed
